### ISSUE TITLE
 The declaration file is missing the jsdoc comment

### ISSUE THREAD

**Issue by kongLau** [wessberg/rollup-plugin-ts#64]
>index.ts
```
export * from "./foo";
```

> foo.ts
```
export class Foo {
    /**
     * m
     * @param {number} arg
     * @return {number} res
     */
    m(arg:number): number {
        return arg;
    }

    /**
     * static m1
     * @param {number} arg
     * @return {number} res
     */
    static m1(arg:number): number {
        return arg;
    }
}
```
>declaration file
```
declare class Foo {
    m(arg: number): number;
    static m1(arg: number): number;
}
export { Foo };
```
Node: v8.10.0
OS: Windows 10 1909
TypeScript: 3.7.4
rollup-plugin-ts: 1.2.1

**Comment by wessberg** [comment]
Hey there. Yes, this is unfortunate. It is related to https://github.com/microsoft/TypeScript/issues/17146. When merging and generating the declarations, nodes are cloned, and currently TypeScript's compiler APIs doesn't offer APIs for transforming JSDoc comments. I've tried my best at working around it, but for the time being we'll need to await movement on the related issue.

**Comment by wessberg** [comment]
Hey there. I've come up with a workaround that effectively resolves the issue. It has been released as part of [v1.2.5](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.2.5)
